clients/src/main/scala/io/treeverse/clients/s3/S3MetaRangeReader.scala

```diff
@@ -0,0 +1,22 @@
+package io.treeverse.clients.s3
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.GetObjectRequest
+import io.treeverse.clients
+import io.treeverse.clients.{MetaRangeReader, SSTableReader}
+
+import java.io.File
+
+class S3MetaRangeReader(var s3: AmazonS3, var bucket: String, var repositoryPrefix: String) extends MetaRangeReader {
```

Comment:
I don't know what's the final purpose of this client, but it *probably* shouldn't be calling S3 directly (i.e. if it's a Hadoop embeddable library, you should be coordinating IO through the HadoopFileSystem